source/fundamentals/stable-api.txt

```diff
@@ -0,0 +1,130 @@
+.. _csharp-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. note::
+
+   The {+stable-api+} feature requires MongoDB Server 5.0 or later.
+
+   You should use the {+stable-api+} feature only if all the MongoDB
+   servers you're connecting to support this feature.
```

Comment:
```suggestion
   You should only use the {+stable-api+} feature if all of the MongoDB
   servers you're connecting to support this feature.
```

Reply:
Agreed on adding "of". I put "only" where I did per [the style guide.](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o)